erplibre_website_snippets_timelines/views/snippets.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+  <!-- Timeline snippets -->
+  <template id="s_timeline_vertical" name="Verticle Timeline">
+    <section class="o_timeline_vertical bg-200 pt16 pb16">
+      <div class="container">
+        <div class="timeline-wrapper">
+          <div class="timeline-inner">
+            <div class="timeline-inner__text">
+              <h4 class="o_h4">Organization Inception</h4>
+              <p class="o_p">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+            </div>
+            <div class="timeline-inner__date">
+              <h3 class="o_h3">JAN</h3>
+              <h6 clas="o_h6">2021</h6>
+            </div>
+          </div>
+        </div>
+        <div class="timeline-wrapper">
+          <div class="timeline-inner">
+            <div class="timeline-inner__text">
+              <h4 class="o_h4">Organization Inception</h4>
+              <p class="o_p">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+            </div>
+            <div class="timeline-inner__date">
+              <h3 class="o_h3">JAN</h3>
+              <h6 clas="o_h6">2021</h6>
+            </div>
+          </div>
+        </div>
+        <div class="timeline-wrapper">
+          <div class="timeline-inner">
+            <div class="timeline-inner__text">
+              <h4 class="o_h4">Organization Inception</h4>
+              <p class="o_p">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+            </div>
+            <div class="timeline-inner__date">
+              <h3 class="o_h3">JAN</h3>
+              <h6 clas="o_h6">2021</h6>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+  </template>
+
+  <template id="s_timeline_horizontal" name="Horizontal Timeline">
+    <section class="o_timeline_horizontal m-3">
+      <div class="container">
+        <div class="timeline-wrapper">
+          <div class="timeline-text">
+            <p class="o_p">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+            <h2>2021</h2>
+          </div>
+          <div class="timeline-icon">
+            <i class="fa fa-2x fa-font-awesome rounded-circle bg-primary s_features_grid_icon"/>
+            <h4>Organization Inception</h4>
+          </div>
+        </div>
+      </div>
+    </section>
+  </template>
+
+  <!-- Add snippet to menu -->
+  <template id="add_snippet_structure_to_menu" inherit_id="website.snippets">
+    <xpath expr="//div[@id='snippet_structure']/div[@class='o_panel_body']" position="inside">
```

Comment:
Error-prone use of @Class in view add_snippet_structure_to_menu (): use the hasclass(*classes) function to filter elements by their classes